_includes/sidebar.html

```diff
@@ -0,0 +1,132 @@
+<!-- detail page sidebar -->
+{% if page.partner_site %}
+{% assign urlparts = page.url | split: "/" %}
+{% assign year = urlparts[1] %}
+{% assign location = urlparts[2] %}
+{% assign sidebar = site.data[year][location].sidebar %}
+  <div class="col-md-4">
+    <ul class="nav flex-column">
+      <li class="nav-item">
+        <a class="nav-link" href="{{page.dir | prepend: site.baseurl}}">Overview</a>
+      </li>
+      {% for nav in sidebar %}
+        <li class="nav-item">
+          <a class="nav-link" href="{{nav.url}}">{{nav.name}}</a>
+        </li>
+      {% endfor %}
+      <li class="nav-item">
+        <a class="nav-link btn btn-dark d-xl-inline-flex mt-3 ml-3" href="apply">Apply</a>
+      </li>
+      <br/>
+    </ul>
+    <ul class="nav flex-column">
+      <a class="nav-link collapsed" onmouseover=""" style="cursor: pointer;" data-toggle="collapse" data-target="">Other Events:</a>
+      {% assign check2018 = site.data.2018[location].hero %}
+      {% assign check2019 = site.data.2019[location].hero %}
+      {% assign check2020 = site.data.2020[location].hero %}
+      {% if check2018 or check2019 or check2020 %}
+        {% if check2018 %}
+          {% if year != '2018' %}
+            <a class="nav-link" href="{{site.baseurl}}/2018/{{location}}">2018</a>
+          {% endif %}
+        {% endif %}
+        {% if check2019 %}
+          {% if year != '2019' %}
+            <a class="nav-link" href="{{site.baseurl}}/2019/{{location}}">2019</a>
+          {% endif %}
+        {% endif %}
+        {% if check2020 %}
+          {% if year != '2020' %}
+            <a class="nav-link" href="{{site.baseurl}}/2020/{{location}}">2020</a>
+          {% endif %}
+        {% endif %}
+      {% endif %}
+    </ul>
+    </ul>
+  </div>
+{% endif %}
+
+<!-- locations sidebar -->
+{% if page.layout == 'locations' %}
+  <ul class="nav flex-column" id="accordionMenu">
+    {% for location in data.locations %}
+      {% assign index_target = forloop.index %}
+      {% assign year = location.year %}
+      <li class="nav-item">
+        <a class="nav-link" onmouseover="" style="cursor: pointer;" data-toggle="collapse" data-target="#collapse{{forloop.index}}">{{year}}</a>
+      </li>
+      <ul class="nav flex-column collapse" id="collapse{{forloop.index}}" aria-labelledby="{{forloop.index}}" data-parent="#accordionMenu">
```

Comment:
this seems to mess up the styles when the accordian menu is transitioning. I think this might be a bootstrap issue, something to do with the `.collapsing` style. https://getbootstrap.com/docs/4.0/components/collapse/#example

Reply:
addressed in: https://github.com/compsocialscience/summer-institute/pull/1240/commits/b997acb2e3f32fb2a09b23af0aa086c207cb6541